code/modules/projectiles/guns/energy/laser.dm

```diff
@@ -231,6 +231,19 @@
 		)
 	twohanded = FALSE
 
+/obj/item/weapon/gun/energy/psychic/mindflayer
+	name = "Prototype: mind flayer"
+	desc = "A cruel weapon designed to break the minds of those it targets, causing sanity loss and mental breakdowns."
+	icon = 'icons/obj/guns/energy/xray.dmi'
+	icon_state = "xray"
+	projectile_type = /obj/item/projectile/beam/psychic
+	fire_sound = 'sound/weapons/Laser.ogg'
+	slot_flags = SLOT_BELT|SLOT_BACK
+	fire_delay = 10
+	price_tag = 3000
```

Comment:
Price may be a tad too high given the amount of matter used

Reply:
I can lower it if you want, this was the base price.

Reply:
Alright, I made it less valuable.